.dockerignore

```diff
@@ -0,0 +1,6 @@
+*
+!*.js
+!src
+!*.json
+!scripts
```

Comment:
in case this ignores scripts, they should be availab le in the deploy!

Reply:
They don't, they ignore everything that is not marked with a !